### ISSUE TITLE
Unable to discover version of typescript used by SST

### ISSUE THREAD

**Issue by jacknewberry** [serverless-stack/sst#2674]
We are using SST deployed via Seed for a typescript API. We have not yet upgraded to SST v2.

I recently updated to typescript 4.9, and started using the new `satisfies` keyword.
// package.json
```json
"devDependencies": {
  "@serverless-stack/cli": "^1.15.11",
  "typescript": "^4.9.5",
```

Updating the version

**Comment by patsuitestudios** [comment]
Yeah also seeing this error. I'm trying to use the new `satisfies` operator from TypeScript 4.9 in my stack definitions (not in my lambda handlers), and getting the same error as @jacknewberry.

**Comment by patsuitestudios** [comment]
For what it's worth, I was able to use TypeScript 4.9 successfully after upgrading to SST v2 👍

**Comment by thdxr** [comment]
We are transition this repo to https://github.com/sst/v2 and are closing this issue for now. If it is still relevant please reopen it there - sorry for the trouble
